content/en/docs/concepts/security/index.md

```diff
@@ -598,14 +598,11 @@ peers:
 
 ## Authorization
 
-Istio's authorization feature - also known as Role-based Access Control (RBAC)
-- provides namespace-level, service-level, and method-level access control for
-services in an Istio Mesh. It features:
-
-- **Role-Based semantics**, which are simple and easy to use.
-- **Service-to-service and end-user-to-service authorization**.
-- **Flexibility through custom properties support**, for example conditions,
-  in roles and role-bindings.
+Istio's authorization feature provides mesh-level, namespace-level, and workload-level
+access control on workloads in an Istio Mesh. It features:
+
+- **Single API**, which is easy to use and maintain.
+- **Flexible semantic**, extra conditions are supported for finer grained access control.
```

Comment:
"extra" and "finer grained" are compared to what?

Reply:
"extra" is compared to the first-class fields provided in the API, "finer grained" comes from the rich semantics in the conditions, like the support of JWT claims, etc.

Reply:
Done.